testcase/cross-region/cross-region.go

```diff
@@ -198,7 +195,7 @@ func (c *crossRegionClient) setupDB() error {
 		return err
 	}
 	for i := 1; i <= 3; i++ {
-		stmt := fmt.Sprintf(stmtPolicy, fmt.Sprintf("p%v", i), fmt.Sprintf("dc-%v", i))
+		stmt := fmt.Sprintf(stmtPolicy, fmt.Sprintf("p%v", i), DCLocations[i-1])
```

Comment:
Why not just start `i` form 0? `i-1` here seems not necessary.

Reply:
Because the partition name are formatting by `fmt.Sprintf("p%v", i)`.